### ISSUE TITLE
Could you please tell what is the metric used for searching the hyperparameters in this paper?

### ISSUE THREAD

**Issue by ZhouJunyu9102** [grammarly/gector#130]
I search the tweak inference hyperparameters using cross entropy, where I got 65.4 f0.5 on CoNLL-2014 test set.
Could you please tell what is the metrics for searching the hyperparameters?
Did you use f0.5 for tuning the hyperparameters?

**Comment by skurzhanskyi** [comment]
Did you find the answer in the paper?
